code/game/objects/items/robot/robot_items.dm

```diff
@@ -74,6 +74,7 @@
 									"<span class='notice'>You playfully boop [M] on the head!</span>")
 					user.do_attack_animation(M, ATTACK_EFFECT_BOOP)
 					playsound(loc, 'sound/weapons/tap.ogg', 50, 1, -1)
+					M.apply_damage(1) //Booping is now harmful.

```

Comment:
Aren't you supposed to define a type of damage, or does it default to brute without